community-spotlight-conference-backend-world-affairs-conference.mdx

```diff
@@ -0,0 +1,92 @@
+
+## A registration flow with optimized session assignment
+
+As registration flows should be, this one was pretty simple overall and leveraged [Lucia Auth](https://lucia-auth.com/) to provide a secure onboarding experience. The form intakes your personal details, as well as lunch preferences and plenary sessions you’re most interested in attending. On submit, the data is then posted to the Xata database and the user is prompted with a security code to confirm their email and identity.
+
+![Input form for conference registration](/images/cs-wac-registration-form.png)
+
+In most cases, this would be the end of the registration flow. Because this is a conference with some pretty popular and well known speakers, Samuel needed to come up with a way to provide the most optimal schedule to attendees. How can they provide personalized conference tracks that are optimized to make the most people happy?
+
+It turns out that the problem of attendee assignment to plenaries is very similar to the problem faced when assigning medical residents to hospitals for their medication. The WAC team ended up using the same algorithm to solve this problem, the [Gale-Shapely algorithm](https://en.wikipedia.org/wiki/Gale%E2%80%93Shapley_algorithm). They created a weighted graph between every single attendee and every single plenary. The weight of the edges are stored in Xata and Samuel and his team created an algorithm to provide an optimal assignment of sessions for every single attendee.
+
+## Customized, performant and scalable administrative dashboard
+
+The other users they had to build for were the administrators of the conference. The WAC administrators had a few very specific requirements.
+
+- An extremely fast table to search, review and edit attendees information and status
+- Trends and analytics on sign ups over time, schools that registered and more
+- A content management system for the website and registration application
+
+In order to quickly find attendees to see if their registrations are complete, they ended up embedding [AG Grid](https://ag-grid.com/) into the admin app. Sveltekit wasn’t supported natively, so Samuel and his team had to create their own integration in order for the two to work well together. What you saw in this view and others depended entirely on the roles and permissions you had as an admin. Having this granular role based access control on the system was important as the number of users grew.
+
+![Attendee administrative table](/images/cs-wac-admin-table.png)
+
+In order to perform interactive analytics on the data itself, the team relied on [Xata aggregations](https://xata.io/docs/sdk/aggregate). This allowed them to chart the number of sign ups over time and segment based on schools and other profile attributes. The view of the data will help future conference administrators with outreach, promotion and further optimization.
+
+![Dashboard for attendee and speaker analytics](/images/cs-wac-admin-dashboard.png)
```

Comment:
Cut out the Alex 😄

Reply:
@Samuel-Martineau / @miukimiu can you cut this out please?